components/card/index.css

```
@@ -176,30 +304,28 @@ governing permissions and limitations under the License.
   justify-content: flex-end;
 }
 
-.spectrum-Card--quiet {
-  .spectrum-Card-preview {
-    min-block-size: var(--spectrum-card-quiet-min-height);
-  }
-}
-
 .spectrum-Card--quiet,
 .spectrum-Card--gallery {
   block-size: 100%;
-  min-inline-size: var(--spectrum-card-quiet-min-width);
+  min-inline-size: var(--mod-card-quiet-minimum-width, var(--spectrum-card-quiet-minimum-width));
   border-width: 0;
   border-radius: 0;
   overflow: visible;
+  border-color: transparent;
+  background-color: transparent;
 
   .spectrum-Card-preview {
+    border-radius: var(--mod-card-corner-radius, var(--spectrum-card-corner-radius));
+    background-color: var(--mod-card-quiet-background-color, var(--spectrum-card-quiet-background-color));
+    min-block-size: var(--mod-card-preview-minimum-height, var(--spectrum-card-preview-minimum-height));
     inline-size: 100%;
     flex: 1;
-    padding: var(--spectrum-card-quiet-preview-padding);
     margin: 0 auto;
     box-sizing: border-box;
-    border-radius: var(--spectrum-card-quiet-border-radius);
     position: relative;
-    transition: background-color var(--spectrum-global-animation-duration-100);
+    /*transition: background-color var(--spectrum-global-animation-duration-100);/* TODO need token */
```

```suggestion
    transition: background-color var(--spectrum-animation-duration-100);
```
I'm not sure when we'll see these in the `@adobe/spectrum-tokens` package, but we do have these available in [our custom tokens](https://github.com/adobe/spectrum-css/blob/e8502f3cfa7547b8b1e3f1dbf8b14083ceb5ba49/components/tokens/custom-spectrum/custom-vars.css#L19-L36).